### PR TITLE
Implement speedup ui updates

### PR DESCRIPTION
**Pull request by rango527**
Implement speedup ui updates

### CODE REVIEW

File: packages/client/src/util/localStorage.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { LiquidityBasketData } from 'types/states';
 
-export const SKIP_DURATION = 240; // 5 sec for test. Should be 240 mins in production
+export const SKIP_DURATION = 14400; // 5 sec for test. Should be 240 mins in production
```
Comment (taariq) [inline packages/client/src/util/localStorage.ts]: @rango527 Any reason why we increased this to 14,400? I

**Review by taariq — CHANGES_REQUESTED** [review]
Curious about the skip duration update. Any reason why that change?

File: packages/client/src/util/localStorage.ts (hunk 1 above)
Comment (rango527) [reply packages/client/src/util/localStorage.ts]: That is the value of 240 mins.
14400 is the value of 4 hours.(in seconds)
Originally there were only mins and seconds.
So if we set it as 4 hours, then it was showed as 240 : 00
I updated it to show hours also, so now it is showing like as 04 : 00 : 00

Comment (taariq) [reply packages/client/src/util/localStorage.ts]: Okay. But we're still testing so please ratched this down back to about 1 minute. We'll continue to increase it as we get ready for contract launch.

Comment (rango527) [reply packages/client/src/util/localStorage.ts]: Got it

**Review by taariq — APPROVED** [review]
LGTM!

**Pull request merged**